sdk/trace/provider.go

```diff
@@ -257,10 +254,15 @@ func (p *TracerProvider) Shutdown(ctx context.Context) error {
 			err = sps.sp.Shutdown(ctx)
 		})
 		if err != nil {
-			return err
+			if retErr == nil {
+				retErr = err
+			} else {
+				// Poor man's list of errors
+				retErr = fmt.Errorf("%v; %v", retErr, err)
+			}
```

Comment:
```suggestion
			if retErr != nil {
				// Poor man's list of errors
				retErr = fmt.Errorf("%v; %v", retErr, err)
			}
			retErr = err
```

Reply:
This doesn't work the way it should, does it?

Reply:
@ash2k's version looks correct to me.

Reply:
```suggestion
			if retErr != nil {
				// Poor man's list of errors
				err = fmt.Errorf("%w; %w", retErr, err)
			}
			retErr = err
```

This might work, but the original also looks correct.  Any reason to not use `multierr` here?

Reply:
@Aneurysm9 If you are talking about https://pkg.go.dev/go.uber.org/multierr, it's not a dependency at the moment and it doesn't feel worth adding it just for this case. In 99% of cases there will be no errors here and it's probably very unlikely that there will be more than one. In this PR I just wanted to ensure all processors are shut down even if there is an error. If there is a better way to handle multiple errors that is discovered later, this code can always be changed.

Reply:
@Aneurysm9 are you satisfied with this answer, or are there further changes you want to request?